source/Calamari.Terraform/DestroyTerraformConvention.cs

```diff
@@ -2,21 +2,25 @@
 using System.Collections.Generic;
 using Calamari.Deployment;
 using Calamari.Integration.FileSystem;
+using Calamari.Integration.Processes;
 
 namespace Calamari.Terraform
 {
     public class DestroyTerraformConvention : TerraformConvention
     {
-        public DestroyTerraformConvention(ICalamariFileSystem fileSystem) : base(fileSystem)
+        readonly ICommandLineRunner commandLineRunner;
+
+        public DestroyTerraformConvention(ICalamariFileSystem fileSystem, ICommandLineRunner commandLineRunner) : base(fileSystem)
         {
+            this.commandLineRunner = commandLineRunner;
         }
 
         protected override void Execute(RunningDeployment deployment, Dictionary<string, string> environmentVariables)
         {
-            using (var cli = new TerraformCLIExecutor(fileSystem, deployment, environmentVariables))
+            using (var cli = new TerraformCLIExecutor(fileSystem, commandLineRunner, deployment, environmentVariables))
             {
-                cli.ExecuteCommand("destroy", "-force", "-no-color", cli.TerraformVariableFiles,
-                    cli.ActionParams);
+                cli.ExecuteCommand("destroy", "-force", "-no-color", cli.TerraformVariableFiles, cli.ActionParams)
+                    .VerifySuccess();
```

Comment:
This particular overload of `ExecuteCommand` called `VerifySuccess`. Moved it out so each of the overloads behaves the same way.